pkg/plugins/models.go

```diff
@@ -126,8 +126,9 @@ type AddedComponent struct {
 }
 
 type AddedFunction struct {
-	Targets []string `json:"targets"`
-	Title   string   `json:"title"`
+	Targets     []string `json:"targets"`
+	Title       string   `json:"title"`
+	Description string   `json:"description"`
 }
```

Comment:
@leventebalogh btw, i think the missing `Description` was a bug that I fixed on the fly here. should i close this PR and reopen a new one to fix this? 
the validator [here](https://github.com/grafana/grafana/blob/de42b991e2aa4e5d68837ff3d6ace2faa9dddf88/public/app/features/plugins/extensions/usePluginFunctions.tsx#L39) is still checking against `description` and without it being set here in model.go, this check will almost always fail.

Reply:
https://github.com/grafana/grafana/pull/107964